cni/network/network.go

```diff
@@ -538,7 +538,7 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 			logAndSendEvent(plugin, fmt.Sprintf("[cni-net] Created network %v with subnet %v.", networkID, ipamAddResult.hostSubnetPrefix.String()))
 		}
 
-		natInfo := getNATInfo(nwCfg.ExecutionMode, options[network.SNATIPKey], nwCfg.MultiTenancy, enableSnatForDNS)
+		natInfo := getNATInfo(nwCfg.ExecutionMode, nwCfg.IPAM.Mode, options[network.SNATIPKey], nwCfg.MultiTenancy, enableSnatForDNS)
```

Comment:
can pass nwCfg pointer instead of individual fields

Reply:
ok